packages/langium/src/documentation/comment-provider.ts

```diff
@@ -0,0 +1,42 @@
+/******************************************************************************
+ * Copyright 2023 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import type { GrammarConfig } from '../grammar';
+import type { LangiumServices } from '../services';
+import type { AstNode } from '../syntax-tree';
+import { findCommentNode } from '../utils/cst-util';
+
+/**
+ * Provides comments for AST nodes.
+ */
+export interface CommentProvider {
+    /**
+     * Defines the token types that are considered as comments.
+     *
+     * The default implementation `DefaultCommentProvider` will return the `multilineCommentRules` from the grammar config.
+     */
+    getCommentTokenTypes(): string[];
```

Comment:
Suggestion: I thought about to use this to replace `GrammarConfig#multilineCommentRules`, but they actually serve different purposes. I'm paddling back on this and propose to remove this interface method.

Reply:
So I guess I have to use `GrammarConfig#multilineCommentRules` in-place